plugins/scaffolder-backend/src/service/rules.ts

```diff
@@ -129,10 +133,45 @@ function buildHasProperty<Schema extends z.ZodType<JsonPrimitive>>({
   });
 }
 
+export const createTaskPermissionRule = makeCreatePermissionRule<
+  SerializedTask,
+  {
+    key: string;
+    values: any;
+  },
+  typeof RESOURCE_TYPE_SCAFFOLDER_TASK
+>();
+
+export const hasCreatedBy = createTaskPermissionRule({
```

Comment:
as catalog already has `isEntityOwner` I would suggest `isTaskOwner`
but up to @backstage/scaffolder-maintainers 

```suggestion
export const isEntityOwner = createTaskPermissionRule({
```

Reply:
Discussed this in the Scaffolder SIG and we decided to go ahead with renaming this to `isTaskOwner` :+1: